olexandr.vakulinskiy/HW2/Task2/extended_string_class.rb

```diff
@@ -0,0 +1,18 @@
+class String
+  def valid?
+    brackets = { "(" => ")", "<" => ">", "{" => "}", "[" => "]" }
+    brackets_stack = []
+    self.each_char do
```

Comment:
`self` [is not required here](https://github.com/rubocop-hq/ruby-style-guide#explicit-self)